SmartPlaces.Facilities/products/Topology/src/Topology.csproj

```diff
@@ -0,0 +1,54 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Worker">
+
+	<PropertyGroup>
+		<OutputType>Exe</OutputType>
+		<TargetFramework>net6.0</TargetFramework>
+		<ImplicitUsings>enable</ImplicitUsings>
+		<Nullable>enable</Nullable>
+		<AssemblyName>$(MSBuildProjectName)</AssemblyName>
+		<RootNamespace>$(MSBuildProjectName)</RootNamespace>
+		<CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)..\..\..\.config\CodeAnalysis.ruleset</CodeAnalysisRuleSet>
+		<Deterministic>true</Deterministic>
+		<LangVersion>10</LangVersion>
+		<RepositoryRootPath>$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)\..'))</RepositoryRootPath>
```

Comment:
Where is this used? Its setting the Repo Root to `SmartFacilities.Places\products\Topology`

Reply:
Was brought over from the common.props, but I don't see a reference to it anywhere.